app/models/user.rb

```diff
@@ -482,6 +485,42 @@ def title
     name
   end
 
+  def number_visits_per_day
+    return user.sign_in_count.round(1) if user.created_at.today?
+
+    (user.sign_in_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
+  end
+
+  def number_articles_read
+    user.page_views.select(:article_id).distinct.count
+  end
+
+  def number_articles_per_day
+    article_count = user.page_views.select(:article_id).distinct.count
+    return article_count.round(1) if user.created_at.today?
+
+    (article_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
+  end
+
+  def average_active_time_per_day
+    seconds_spent_reading = user.page_views.sum(:time_tracked_in_seconds).round(1)
```

Comment:
here we sum the `time_tracked_in_seconds` attributes of all the `page_views` associated with the current `user`